lib/src/handler/json_wire/timeouts.dart

```diff
@@ -47,4 +48,16 @@ class JsonWireTimeoutsHandler extends TimeoutsHandler {
   void parseSetPageLoadTimeoutResponse(WebDriverResponse response) {
     parseJsonWireResponse(response);
   }
+
+  @override
+  WebDriverRequest buildGetTimeoutsRequest() {
+    // Not supported by JSON protocol:
+    // https://www.selenium.dev/documentation/legacy/json_wire_protocol/#command-summary
+    throw UnimplementedError();
```

Comment:
Do we *plan* to support it? 
Otherwise use `UnsupportedError`.

Reply:
I am not aware of plans to support getting timeouts in the JsonWire webdriver API. Switched to `UnsupportedError`.